components/brave_new_tab_ui/components/default/braveToday/cards/_articles/cardArticleLarge.tsx

```diff
@@ -39,7 +39,7 @@ const promotedInfoUrl = 'https://brave.com/brave-today'
 function onClickPromoted (e: React.MouseEvent) {
   const openInNewTab = e.ctrlKey || e.metaKey
   if (openInNewTab) {
-    document.open(promotedInfoUrl, '__blank')
+    document.open(promotedInfoUrl, '__blank', 'noreferrer')
```

Comment:
@petemill is this ok? with 3 arguments this behaves like window.open

Reply:
see https://developer.mozilla.org/en-US/docs/Web/API/Document/open which implies the two-argument window.open form is just like calling `document.open()`, i'm not sure that's what you want